snippets/base/admin/adminmodels.py

```diff
@@ -1275,3 +1275,78 @@ def has_change_permission(self, request, obj=None):
 
     def has_delete_permission(self, request, obj=None):
         return False
+
+
+class DailySnippetsMetrics(admin.ModelAdmin):
```

Comment:
Typo, this should be `DailySnippetMetrics` since everything else is also singular. This inconsistency made to make an error in `ADMIN_REORDER` and the model should be of course `DailySnippetMetrics` too